syft/generic/frameworks/hook/hook.py

```diff
@@ -187,9 +186,9 @@ def _which_methods_should_we_auto_overload(self, tensor_type: type):
             A list of methods to be overloaded.
         """
 
-        boolean_comparators = ["__gt__", "__ge__", "__lt__", "__le__"]
+        self.boolean_comparators = ["__gt__", "__ge__", "__lt__", "__le__"]
```

Comment:
Maybe it would be better to put this in `__init__`?

Reply:
Correct! I'll set it globally, but I can't put it in init as it is an abstract method